apps/manifest-demo/3010-rspack-provider/src/App.tsx

```diff
@@ -1,4 +1,6 @@
 import LocalButton from './Button';
+import { Await } from 'react-router-dom';
+console.log(Await);
```

Comment:
the test log can be removed

Reply:
this is necessary, it can test import shared in prefetch file